apps/edxapp/templates/services/nginx/static-svc.yml.j2

```
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: edxapp
+    service: nginx
+    deployment_stamp: "{{ deployment_stamp }}"
+    service_prefix: "{{ prefix }}"
+    type: static-service
+    removable: "no"
```

Why not using a boolean here?
```suggestion
    removable: false
```

---

Kubernetes label values must be strings, so we can't use boolean values directly in yaml definitions. With your suggestion, it triggers the following error : 

> v1.Service.ObjectMeta: v1.ObjectMeta.Labels: ReadString: expects \\\\\" or n, but found f, error found in #10 byte of

As an alternative, we can use the following syntax : 

```yaml
removable: "false"
```

But I think this ambiguity in types increases the risk of errors, that's why I preferred to use "yes"/"no" values instead.

---

Ok, then. Let's keep it as is.